lib/code_corps/emails/receipt_email.ex

```diff
@@ -3,9 +3,15 @@ defmodule CodeCorps.Emails.ReceiptEmail do
   import Bamboo.PostmarkHelper
 
   alias CodeCorps.Emails.BaseEmail
-  alias CodeCorps.{DonationGoal, Project, Repo, StripeConnectCharge, StripeConnectSubscription, WebClient}
+  alias CodeCorps.{DonationGoal, Project, Repo, StripeConnectCharge, StripeConnectSubscription, WebClient, User}
 
-  @spec create(StripeConnectCharge.t, Stripe.Invoice.t) :: Bamboo.Email.t
+  @spec get_name(User.t) :: String.t
+  def get_name(%User{ first_name: nil }), do: "there"
+
+  @spec get_name(User.t) :: String.t
+  def get_name(%User{ first_name: name}), do: name
```

Comment:
I think you can condense the spec to just above the first line and then not separate the lines between these.

Reply:
I also wonder whether we should place this fn into the `BaseEmail` so that it's available to use across other emails.

Reply:
@joshsmith  if the functionality is needed in other emails the function can live in BaseEmail